spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JSqlParserQueryEnhancer.java

```diff
@@ -221,7 +222,7 @@ private Set<String> getJoinAliases(String query) {
 		}
 
 		Select selectStatement = (Select) statement;
-		if (selectStatement.getSelectBody()instanceof PlainSelect selectBody) {
+		if (selectStatement.getSelectBody() instanceof PlainSelect selectBody) {
 			return getJoinAliases(selectBody);
```

Comment:
Whereas I sympathize with the desire to introduce this space WHICH SHOULD BE HERE, our current tools, due to `getSelectBody()` (the parentheses) will take that space back out the next time I'm inside this class and I reformat it. Hence, I'd rather not introduce that churn.